README.md

```diff
@@ -7,7 +7,7 @@
 
 > stop using `npm version`, use `standard-version` it rocks!
 
-Automatic versioning and CHANGELOG management, using GitHub's new squash button and
+NPM and Cordova Automatic versioning and CHANGELOG management, using GitHub's new squash button and
```

Comment:
Rather than changing this copy, lets a section towards the end of the the README (below Badges!) that's something along the lines of:

## Cordova Support

standard-version works with Cordova, and will automatically increment `android-versionCode` in your config.xml.